.git-blame-ignore-revs

```diff
@@ -1,3 +1,4 @@
 # Dusting
 c380d69ba5f0813410af5eba98c3d3b83e4c288f
 b7707e028a9838565a51d617382b0f2690bcd5cb
+a951541cdd4fb5f2ba5dcdfdd07b2a206e707ebd
```

Comment:
Can we keep this in the history, actually? There's only one dusting change and the other stuff should show up in the diffs.

Reply:
I reverted that commit, so I think that solves what you're wanting.